### PR TITLE
Document the deletion_allowed transform flag

### PR DESCRIPTION
**Pull request by sgmiller**

### CODE REVIEW

File: website/content/api-docs/secret/transform.mdx [hunk 1]
```diff
@@ -181,7 +181,9 @@ configuration endpoints, and will be removed in a future release.
   A role using this transformation must exist in this list in order for
   encode and decode operations to properly function.
 
--
+- `deletion_allowed` `(bool: false)` -
+  If true, this transform can be deleted. If true, this transform can be deleted.
```
Comment (benashz) [inline website/content/api-docs/secret/transform.mdx:185]:
```suggestion
  If true, this transform can be deleted.
```

**Review by benashz — APPROVED** [review]
minor nit, after that +1

**Review by schultz-is — APPROVED** [review]
lgtm

File: website/content/api-docs/secret/transform.mdx (hunk 1 above)
Comment (sgmiller) [reply website/content/api-docs/secret/transform.mdx:185]: Thanks.

**Pull request merged**